tests/tracing/test_provider.py

```diff
@@ -62,3 +63,47 @@ def test_fn():
     assert len(TRACE_BUFFER) == 1
     assert isinstance(_get_tracer(__name__), trace.Tracer)
     TRACE_BUFFER.clear()
+
+
+@pytest.mark.parametrize("enabled_initially", [True, False])
+def test_trace_disabled_context_manager(enabled_initially):
+    if not enabled_initially:
+        mlflow.tracing.disable()
+    assert _is_enabled() == enabled_initially
+
+    @mlflow.trace
+    def test_fn():
+        pass
+
+    with trace_disabled():
+        test_fn()
+        assert len(TRACE_BUFFER) == 0
+        assert not _is_enabled()
+
+    # Recover the initial state
+    assert _is_enabled() == enabled_initially
+
+
+def test_is_enabled():
+    # Before doing anything -> tracing is considered as "on"
```

Comment:
Does "on" mean enabled?

Reply:
Yes!

Reply:
Can we use enabled/disabled instead of on/off?